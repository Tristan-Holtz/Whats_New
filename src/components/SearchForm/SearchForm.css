.nav {
  align-items: left;
  display: flex;
  background: #141527;
  border-bottom: 2px solid #fff;
  height: 60px;
}
