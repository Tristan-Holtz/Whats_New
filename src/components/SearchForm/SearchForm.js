import React from 'react';
import './SearchForm.css';

const SearchForm = props => {
  return (
    <nav className="nav">
      <input className="nav-search"></input>
      <button onClick={props.runSearch}>SEARCH</button>
    </nav>
  );
};

export default SearchForm;
