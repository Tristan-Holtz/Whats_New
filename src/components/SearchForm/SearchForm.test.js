import React from 'react';
import { shallow } from 'enzyme';
import SearchForm from './SearchForm';

describe('SearchForm', () => {
  it('renders a nav', () => {
    const wrapper = shallow(<SearchForm />);

    expect(wrapper).toMatchSnapshot();
  });
});
