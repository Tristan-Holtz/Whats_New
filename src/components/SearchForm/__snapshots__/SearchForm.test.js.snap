// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders a nav 1`] = `ShallowWrapper {}`;
